### DISCUSSION TITLE
osu! crashes with STACK_OVERFLOW error

### DISCUSSION THREAD

**Discussion by Wieku** [ppy/osu#15379] [Q&A] [answered]
Hi!
I'm trying to build the latest lazer master (HEAD is 53afed19ec9d0a8df6692d4138b5c182ea5082c0) and it keeps crashing with code -1073741571 (0xC00000FD)

I've been getting only that code while trying to run lazer through Rider so I decided to run built executable with terminal and I got this:
[lazercrash.txt](https://github.com/ppy/osu/files/7450075/lazercrash.txt)

It looks like it loops at `SkinManager.ComputeHash`.
After adding a print before those lines
https://github.com/ppy/osu/blob/53afed19ec9d0a8df6692d4138b5c182ea5082c0/osu.Game/Skinning/SkinManager.cs#L183-L187

I got this:
![image](https://user-images.githubusercontent.com/5307179/139607466-0e05c1fb-b681-4b91-b018-56ae1be23fbd.png)

This is the skin that breaks it (for some reason github rejected the file even if I changed extension to zip): https://img.wieku.me/21/-%20YUGEN%20FINAL%20WS%20-.osk

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15380
